kn/fotos/entities.py

```diff
@@ -408,6 +409,12 @@ def update_metadata(self, parent, save=True):
 
         return True
 
+    def set_rotation(self, rotation, save=True):
+        self.rotation = rotation
+        self._data['caches'] = [] # invalidate cache
```

Comment:
Dit is wellicht wat ongericht – de exif cache (bestaat die nog?) is bijvoorbeeld nog prima.